src/commands/serve.rs

```diff
@@ -156,11 +156,53 @@ async fn serve(req: fpm::http::Request) -> fpm::Result<fpm::http::Response> {
     } else {
         // url is present in config or not
         // If not present than proxy pass it
+
+        let req_method = req.method().to_string();
         let mut config = fpm::time("Config::read()").it(fpm::Config::read(None, false, Some(&req))
             .await
             .unwrap()
             .set_request(req));
 
+        // if start with -/ and mount-point exists so send redirect to mount-point
+        // We have to do -/<package-name>/remaining-url/ ==> (<package-name>, remaining-url) ==> (/config.package-name.mount-point/remaining-url/)
+        // Get all the dependencies with mount-point if path_start with any package-name so send redirect to mount-point
+
+        let package_dependencies = config.package.deps_contains_mount_point();
+        // if any package name starts with package-name to redirect it to /mount-point/remaining-url/
+
+        if req_method.as_str() != "POST" {
```

Comment:
Invert this, only GET should be redirected, no other method. Also ensure the GET request is not coming from ftd.js (see how to detect if we are called from Ajax: https://stackoverflow.com/questions/63629935/django-3-1-and-is-ajax)